packages/docs/docs/sdk/Kotlin/kotlin/io.logto.sdk.core.constant/-claim-name/-a-t_-h-a-s-h.md

```diff
@@ -0,0 +1,6 @@
+# AT_HASH
+
+//[kotlin](../../../)/[io.logto.sdk.core.constant](../)/[ClaimName](index.md)/[AT_HASH](-a-t_-h-a-s-h.md)
+
+[jvm]\
+const val [AT_HASH](-a-t_-h-a-s-h.md): [String](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-string/index.html)
```

Comment:
The file name seems weird. Is this on purpose?

Reply:
The `-a` represents `A` and the `a` represents `a` because the URI is not case sensitive here.
This file is auto-generated by kotlin.

Reply:
ok

Reply:
Are those auto-generated by a third-party library? Looks so weird.

Reply:
Yes

Reply:
Please refer to this PR: https://github.com/Kotlin/dokka/issues/2229
Maybe someday we can provide a PR for kotlin/dokka. 🤓
@simeng-li

Reply:
exactly what I am feeling, should be configurable.